src/color.js

```diff
@@ -0,0 +1,18 @@
+const colorTasks = (selectedList) => {
+  const todos = [...document.querySelectorAll('.todo')];
+  const taskDivcolor = [...document.querySelectorAll('.task')];
+  for (let i = 0; i < todos.length; i += 1) {
+    for (let i = 0; i < selectedList.tasks.length; i += 1) {
+      if (selectedList.tasks[i].priority === 'High') {
+        taskDivcolor[i].style.background = '#ed1250';
+      } else if (selectedList.tasks[i].priority === 'Medium') {
+        taskDivcolor[i].style.background = '#d3d00f';
+      } else {
+        taskDivcolor[i].style.background = '#0fc53d';
+      }
```

Comment:
- [ ] As stated in the previous review kindly move all CSS styling to the stylesheet then toggle the styles of the DOM element by toggling the classes of the elements. Please do this for **all** styles in the js modules.